libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -201,6 +201,10 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
         const dialogBody = this._elRef.nativeElement.parentElement;
         let retVal = 0;
         if (dialogBody.tagName.toLowerCase() === 'fd-dialog-body') {
+            const dialogBodyStyle = this._elRef.nativeElement.parentElement.style;
```

Comment:
This can be done with @Inject(DialogBodyComponent), or with some providers, which might be more feasible here.
After you inject parent component, you can call parent component's method to set additional class to the host, which will apply this styling changes, instead of doing this with JS.

Reply:
Great suggestion, I will use `@Optional` to inject it into the constructor